man/fi_rxd.7.md

```diff
@@ -16,23 +16,22 @@ emulated over a base DGRAM provider.
 
 # SUPPORTED FEATURES
 
-The RxD provider currently supports *FI_MSG*, *FI_TAGGED* and *FI_RMA*
-capabilities. It requires the base DGRAM provider to support *FI_MSG*
+The RxD provider currently supports *FI_MSG* capabilities.
+It requires the base DGRAM provider to support *FI_MSG*
```

Comment:
nit: fi_msg support is required of any provider, but the dgram provider in particular.